lib/models/user/notifications.js

```diff
@@ -133,11 +138,25 @@ module.exports = function (UserSchema) {
         }, function (err, messages) {
             if (err) return callback(err);
 
+            var mail = new sgHelper.Mail(
+                new sgHelper.Email(messages.email.from),
+                messages.email.subject,
+                new sgHelper.Email(messages.email.to),
+                new sgHelper.Content('text/html', messages.email.html));
+
+            var sgRequest = mailer.emptyRequest({
+              method: 'POST',
+              path: '/v3/mail/send',
+              body: mail.toJSON()
+            });
+
             // function to send an email notification
             var sendEmail = function () {
-                mailer.send(messages.email, function (err) {
+                mailer.API(sgRequest, function (err) {
                     // errors are SILENTLY logged and ignored
-                    if (err) console.error(err);
+                    if (err) {
+                        console.log("Error sending email: ", err);
```

Comment:
Is there a better option than `console.log` ?

Reply:
Good question; I thought the same thing. I checked and the Amida logger isn't implemented in this repo, and `console.log` is used in other places, which is why I used it.

I wish I would have discovered this sooner. We could have implemented the Amida logger in this repo.

Reply:
I'll make a ticket for that--though it won't be getting in at this time.